applications/tari_dan_wallet_daemon/src/handlers/accounts.rs

```diff
@@ -680,11 +681,13 @@ pub async fn handle_claim_burn(
         new_account: None,
     });
 
-    let finalized = wait_for_result(&mut events, tx_id).await?;
+    let mut finalized = wait_for_result(&mut events, tx_id).await?;
     if let Some(reject) = finalized.finalize.result.reject() {
         return Err(anyhow::anyhow!("Fee transaction rejected: {}", reject));
     }
     if let Some(reason) = finalized.transaction_failure {
+        finalized.finalize.result =
```

Comment:
The transaction result types are a bit of a mess since fee instructions were added (my bad). I think that the execution engine should return this enum variant (maybe `FeeAccept` and `FullAccept` idk) rather than the wallet modifying things. Then we should be able to remove the "weird" `transaction_result` field entirely.

Reply:
Yeah, I agree with that. I don't mind `FeeAccept` and `FullAccept`, but here are some other suggestions:

`FailedButCharged`
`FailedButFeesDeducted`
`OnlyFeesAccept`
`AcceptWithFailures`